chassis/configuration/configuration/reporter/session.comparison.conf.xml

```diff
@@ -51,21 +50,6 @@
         <property name="databaseService" ref="databaseService"/>
     </bean>
 
-    <bean id="workloadFeatureComparator"
-          class="com.griddynamics.jagger.engine.e1.sessioncomparation.workload.WorkloadFeatureComparator">
-        <property name="workloadDecisionMaker">
-            <bean class="com.griddynamics.jagger.engine.e1.sessioncomparation.workload.ThroughputWorkloadDecisionMaker">
```

Comment:
ThroughputWorkloadDecisionMaker - why is it not deleted from the code base as well? It is not used any more

\+ WorkloadDecisionMaker
\+ StdDevMonitoringParameterDecisionMaker


please, review the whole:
/chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/sessioncomparation/monitoring